cypress/integration/mainpage.spec.js

```diff
@@ -53,14 +53,28 @@ beforeEach(() => {
   it('Should be able to type into the search input and see that value in the start point input', () => {
     cy.get('.start-point').type('6245 Garrison St, Arvada, CO 80004')
         .should('have.value', '6245 Garrison St, Arvada, CO 80004')
+        .wait(2000)
```

Comment:
Nice, looks like we had been failing this test - setting a timeout is a good idea